### ISSUE TITLE
Feature Request: Adding the output panel to Gtuner air

### ISSUE THREAD

**Issue by Scachi** [J2Kbr/GtunerAir#16]
By Forum user: XimCo 

Feature Request: Adding the output panel to Gtuner air

https://www.consoletuner.com/forum/viewtopic.php?f=25&t=17222&p=107865&hilit=gtuner+air+output+panel#p107688

https://www.consoletuner.com/forum/viewtopic.php?f=25&t=17222&p=107865&hilit=gtuner+air+output+panel#p107821

J2kbr's reply:
```
Thank you for your input and considerations. Add the output panel to Gtuner Air is something feasible and relatively easy to implement. We can have this as starting point for then see how evolve to something more graphical.
```

**Comment by cwiggy78** [comment]
What a fantastic feature request. This would allow users of the T2 to visually monitor gamepack or script states whilst in game.  So what weapon slot is loaded and parameters for that slot.  Maybe even in the GTuner air GUI set weapon names to be displayed or alternatively set all that extra info within the script or gamepack  and GTuner air displays the info as it's triggered within script or slot.  The possibilities seem endless with this.  Make use of everyone having a mobile phone display next to them and the T2.  Really excited about this J2Kbr.  Please develop this feature.

**Comment by J2Kbr** [comment]
Feature implemented on Gtuner Air 1.22, to be released soon.
